src/app.js

```diff
@@ -1,21 +1,41 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+
+const PORT = process.argv[2] || 8080;
+
+http.createServer((req, res) => {
+  const normalizedUrl = new URL(req.url, `http://${req.headers.host}`);
+  const doesFilePartExist = normalizedUrl.pathname.split('/')[1] === 'file';
+  const partialPathToFile = normalizedUrl.pathname
+    .split('/')
+    .slice(2)
+    .filter(Boolean);
+  const pathToFile = partialPathToFile.length === 0
+    ? 'public/index.html'
+    : ['public', ...partialPathToFile].join('/');
+
+  if (!doesFilePartExist) {
+    res.statusMessage = 'your request should'
+      + ' look in following way /file/index.html';
+
+    res.end(
+      'your request should look in following way /file/index.html,'
+      + ' use part /file/',
```

Comment:
Nope
Your request should return index.html in the next cases
![image](https://user-images.githubusercontent.com/57747481/219409458-ce27c890-f01a-473d-9fec-42695c4f3803.png)
![image](https://user-images.githubusercontent.com/57747481/219409479-1f38d03b-e046-492b-8434-def65379ebc3.png)
![image](https://user-images.githubusercontent.com/57747481/219409493-3182e5e5-ebb2-4a80-8ea6-d29a37f46649.png)